lib/Parser/OglParser.php

```diff
@@ -99,7 +99,7 @@ public function parseStream($handle, \Mesamatrix\Git\Commit $commit = null) {
                         $matrix->addGlVersion($glVersion);
                     }
 
-                    $line = $this->parseSection($glVersion, $matrix, $commit, $handle, $allSupportedDrivers);
+                    $line = $this->parseSection($glVersion, $matrix, $commit, $handle);
```

Comment:
OK, after a closer look, you're right, `$allSupportedDrivers` shouldn't be passed to this line, it's not even declared outside the first `if` block. Gosh I hate PHP for this kind of programming nonsense...